doc/articles/controls/ListViewBase.md

```diff
@@ -7,51 +7,52 @@ list view on each platform for high performance. This document explains how Uno'
 
 This is a stripped-down view of the default style for ListView in Uno:
 
-```xml
-	<!-- Default style for Windows.UI.Xaml.Controls.ListView -->
-	<xamarin:Style TargetType="ListView">
-		<Setter Property="ItemsPanel">
-			<Setter.Value>
-				<ItemsPanelTemplate>
-					<ItemsStackPanel Orientation="Vertical" />
-				</ItemsPanelTemplate>
-			</Setter.Value>
-		</Setter>
-		<Setter Property="Template">
-			<Setter.Value>
-				<ControlTemplate TargetType="ListView">
-					<Border>
-						<ScrollViewer x:Name="ScrollViewer"
-									  xamarin:Style="{StaticResource ListViewBaseScrollViewerStyle}">
-							<ItemsPresenter/>
-						</ScrollViewer>
-					</Border>
-				</ControlTemplate>
-			</Setter.Value>
-		</Setter>
-	</xamarin:Style>
+``` XAML
```

Comment:
This change loses the syntax highlighting. XML is the most suitable identifier to use.

~~~suggestion
```xml
~~~

Reply:
I've just tried and nothing changes. Highlighting remains the same. How do you see it?

Reply:
Do you mean it does not work on the docs site? As on the GitHub it remains the same highlighted

Reply:
![image](https://user-images.githubusercontent.com/2365592/134164640-034f54bf-5963-4cb9-8fa5-5f72017a108e.png)

Reply:
Yeah I mean on the docs site. The docs site depends on highlight.js library which doesn't support XAML as an identifier.

Reply:
That's correct, we cannot merge this PR because the actual target of the documentation (docfx) does not support XAML.